inference/core/managers/pingback.py

```diff
@@ -44,7 +44,9 @@ def __init__(self, manager):
             manager (ModelManager): Reference to the model manager object.
         """
         try:
-            self.scheduler = BackgroundScheduler()
+            self.scheduler = BackgroundScheduler(
```

Comment:
Coalescing subsequent runs to prevent spam logging 
<img width="393" alt="Screenshot 2024-07-25 at 11 11 32" src="https://github.com/user-attachments/assets/4bd790e7-3304-4011-9133-eb79f4da2461">